src/docfx/config/ContributionConfig.cs

```diff
@@ -37,5 +39,10 @@ internal class ContributionConfig
         /// It should be an absolute url or a relative path
         /// </summary>
         public readonly string GitCommitsTime = string.Empty;
+
+        /// <summary>
+        /// The excluded contributors which you do want to show
```

Comment:
do or do not?

Reply:
typo, thanks!

---
In reply to: [210815759](https://github.com/dotnet/docfx/pull/3224#discussion_r210815759) [](ancestors = 210815759)